devfiles/dotnet/devfile.yaml

```diff
@@ -44,3 +47,28 @@ commands:
         component: dotnet
         command: "dotnet run"
         workdir: ${CHE_PROJECTS_ROOT}/dotnet-web-simple
+  -
+    name: build
+    actions:
+    - type: exec
+      component: netcoredbg
```

Comment:
Since in both cases it's building into `/projects` (mounted to all containers) and using the resulting `obj/Debug` folder, can this command be attached to the main ws container? As it is, it's hidden in the normally collapsed "plugins" tab in the "my workspace" sidebar.

Reply:
Makes sense.  I've added an improvement.